sdk/python/symbolchain/nem/SharedKey.py

```diff
@@ -0,0 +1,17 @@
+import sha3
+
+from ..external import ed25519
+from ..SharedKey import SharedKey as BasicSharedKey
+
+
+class SharedKey(BasicSharedKey):
+	@staticmethod
+	def derive_shared_key(key_pair, other_public_key):
+		return BasicSharedKey._derive_shared_key(key_pair, other_public_key, b'nem-nis1')
```

Comment:
probably would go with just nem

Reply:
wanted to make it 8b in length same as 'catapult'